app/src/main/res/values/strings.xml

```diff
@@ -1790,8 +1800,10 @@
     <string name="top_sites_max_limit_content_2">To add a new top site, remove one. Touch and hold the site and select remove.</string>
     <!-- Confirmation dialog button text when top sites limit is reached. -->
     <string name="top_sites_max_limit_confirmation_button">OK, Got It</string>
+    <!-- Label for the show most visited top sites preference -->
+    <string name="top_sites_toggle_top_frecent_sites2">Show most visited top sites</string>
```

Comment:
🚨 **Blocking change** I will have to insist on the **_2** since that is what I am overwhelmingly seeing us do in the file, and I rather keep us consistent with such naming conventions.

```suggestion
    <string name="top_sites_toggle_top_frecent_sites_2">Show most visited top sites</string>
```

Reply:
👌🏽